src/components/image-wrapper/image-wrapper.js

```diff
@@ -0,0 +1,6 @@
+import './image-wrapper.css';
+export const ImageWrapper = ({ children }) => {
```

Comment:
-As I can see the Image wrapper is used for icons an links. In general this is a good idea. But semantically the name is not suitable. How about a component name as IconWithLink?

Another optimizations would be not repeating the usage. To do so this component can be changed to this

```javascript
const IconWithLink = ({icon, link}) => {
const IconComponent = icon;
return <a href={link} className="icon-wapper"><IconComponent /></a>
}

```

Therefore the usage would be like

```javascript
<IconWithLink link="https://somelink.com" icon={FaIntagram} />
```
What do you think?